docs/developer/instruction/autoreduction.rst

```diff
@@ -0,0 +1,58 @@
+How to Modify an Instrument Autoreduction Configuration Page
+==============================
+
+.. note::
+    This document is a draft. If you use the instructions, please consider correcting/improving them.
+
+Instrument scientists can set up a form where Web Monitor users with staff status can modify the autoreduction script for the instrument.
+The web form lets the instrument scientists change parameter values in the autoreduction script and create a new version on the analysis cluster at
+
+``/<facility (SNS or HFIR)>/<instrument>/shared/autoreduce/reduce_<instrument>.py``
```

Comment:
I don't know that we need the suggested facilities. If you want to keep them give examples outside of the file path
```suggestion
``/<facility>/<instrument>/shared/autoreduce/reduce_<instrument>.py``
```
I don't know if this would format better as a code-block